File: 2436-make-array-zero-by-subtracting-equal-amounts/2436-make-array-zero-by-subtracting-equal-amounts.java
class Solution {
    public int minimumOperations(int[] nums) {
        HashSet<Integer> Hash = new HashSet<>();

        for(int num : nums){
          if(num != 0){
            Hash.add(num);
          }
                
            
        }

        return Hash.size();
        
    }
}

File: 2436-make-array-zero-by-subtracting-equal-amounts/2436-make-array-zero-by-subtracting-equal-amounts.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minimumOperations = function(nums) {
    let set = new Set();
    for(let num of nums){
        if(num !== 0){
            set.add(num);
        }
    }
    return set.size;
};